projects/packages/my-jetpack/_inc/components/golden-token/tooltip/index.jsx

```diff
@@ -0,0 +1,70 @@
+import { Button, JetpackIcon } from '@automattic/jetpack-components';
+import { Popover } from '@wordpress/components';
+import { dateI18n } from '@wordpress/date';
+import { __, sprintf } from '@wordpress/i18n';
+import classNames from 'classnames';
+import { useCallback, useState } from 'react';
+import styles from './style.module.scss';
+import './style.global.scss';
+
+/**
+ * Golden Token Tooltip.
+ *
+ * This is a rather quick port of the <IconTooltip> Jetpack component.
```

Comment:
Personally I don't mind the the duplication. This is a special token and it deserves a special component 😂 at least for now.